src/Core/Engine/Renderer/Renderer.hpp

```
@@ -100,15 +110,19 @@ class Renderer : public IService {
     void Submit(SubmitObject& submit);
     bool PrepareObject(RenderObject& object);
-
     void DrawObject(RenderObject& object, size_t InstanceCount);
+
     void ApplyState();
+
     void RenderSceneToFBO();
+    void PresentTexture(GLuint texture);
     void PresentScene();
-    void Init();
+    GLuint RunPostProcessChain(GLuint inputTexture);
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)